docs/application/dotnet/guides/nui/nui-components/LottieAnimationView.md

```diff
@@ -0,0 +1,55 @@
+# LottieAnimationView
+LottieAnimationView is a common component that is used to show an vector animation. It parses Adobe After Effects animations exported as json with Bodymovin and renders them.
+
+![LottieAnimationView](./media/lottie.gif)
+
+## Create with Property
+
+To create a LottieAnimationView using property, follow these steps:
+
+1. Create LottieAnimationView using the default constructor:
+
+    ```cs
+    lottieAnimationView = new LottieAnimationView();
+    ```
+
+2. Set the properties:
+
+    ```cs
+    lottieAnimationView.Size = new Size(100, 108);
+    lottieAnimationView.URL = "myLottie.json"
+    lottieAnimationView.LoopCount = 3;
+    lottieAnimationView.StopBehavior = StopBehaviorType.CurrentFrame;
+    lottieAnimationView.CurrentFrame = 20;
+
+    lottieAnimationView.SetMinMaxFrame(10, 30);
+
+    parent.Add(lottieAnimationView);
+    ```
+
+## Control animation playback
+After the view is created, you can control its playback.
```

Comment:
After the view is created, you can control its playback:

Reply:
Done